data/json/items/book/fabrication.json

```
@@ -348,7 +348,7 @@
     "skill": "fabrication",
     "required_level": 5,
     "max_level": 10,
-    "proficiencies": [ { "proficiency": "prof_bladesmith", "time_factor": 0.75, "fail_factor": 0.75, "include_prereqs": true } ],
+    "proficiencies": [ { "proficiency": "prof_bladesmith", "time_factor": 0.2, "fail_factor": 0.2, "include_prereqs": true } ],
```

```suggestion
    "proficiencies": [ { "proficiency": "prof_bladesmith", "time_factor": 0.2, "fail_factor": 0.3, "include_prereqs": true } ],
```
Description + other stats.

---

Include_prereqs means it will apply its bonuses to lower level proficiencies if you lack the higher level ones. I wouldn't figure it too much into the balance equations although it does make a book a better proficiency reference